specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/definitions.json

```diff
@@ -4584,6 +4584,77 @@
       ],
       "description": "Policy contract Properties."
     },
+    "PolicyWithComplianceCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyWithComplianceContract"
+          },
+          "description": "Policy Contract value."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "The response of the list policy operation."
+    },
+    "PolicyWithComplianceContract": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyWithComplianceContractProperties",
+          "description": "Properties of the Policy."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
```

Comment:
>/ [](http://example.com/codeflow?start=71&length=1)

Can you please use the v5 definitions as explained here : 

[https://github.com/Azure/azure-rest-api-specs/pull/24720/checks?check_run_id=17178045864](https://github.com/Azure/azure-rest-api-specs/pull/24720/checks?check_run_id=17178045864)